### PR TITLE
BOM -with-drools

### PR DESCRIPTION
**Pull request by marcobattaglia**
This pull request is related to another request pulled in jboss-as-quickstarts: "cdi-portable-extension-drools". In that project is possible to simplify its pom usign this BOM. These BOM links dependencies of jboss-drools-expert. There is no references to others jboss drools projects like Planner, JBPM, fusion.

### CODE REVIEW

**Comment by marcobattaglia** [comment]
Hi Rafael, I have changed the name to jboss-javaee-6.0-with-drools and I have added the dependency of drools planer. 
I've corrected 'witot h Security'. I'm not expert about pom, so it is not  clear for me how to test this and how to integrate this with the original project that is under jboss quickstarts: cdi-portable-extensions-drools as pull request.

File: jboss-javaee-6.0-with-drools/pom.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                <groupId>org.jboss.spec</groupId>
+                <artifactId>jboss-javaee-6.0</artifactId>
+                <version>${version.org.jboss.spec.jboss.javaee.6.0}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+		<dependency>
+			<groupId>org.drools</groupId>
+			<artifactId>knowledge-api</artifactId>
+			<version>${version.jboss.drools}</version>
```
Comment (rafabene) [inline jboss-javaee-6.0-with-drools/pom.xml]: This property definition is not present on your PR. That's why you need to update this file also: https://github.com/jboss-jdf/jboss-bom/blob/master/pom.xml to inform the right ${version.jboss.drools} property. Check the comment:

**Comment by rafabene** [comment]
To test this BOM you will run 'mvn install' on the jboss-bom reposotory. This will install the SNAPSHOT version to your LOCAL only repository. So you can update your cdi-portable-extensions-drools pom.xml to include this BOM under the dependencyManagement section and remove all drools information from cdi-portable-extensions-drools pom.xml (The drools information will come from this BOM). I'll update the CONTRIBUTING.md with this information. More info: JDF-284. If you have any doubts you can ask me or ask @sgilda  who is working with me in this case.

File: jboss-javaee-6.0-with-drools/pom.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    <artifactId>jboss-javaee-6.0-with-drools</artifactId>
+    <packaging>pom</packaging>
+
+    <name>JBoss Java EE 6 Specification APIs JBoss Drools</name>
+    <description>Dependency Management for Java EE 6 Specification APIs with JBoss Drools</description>
+
+    <licenses>
+        <license>
+            <name>GNU Lesser General Public License, Version 2.1</name>
```
Comment (rafabene) [inline jboss-javaee-6.0-with-drools/pom.xml]: Marco, please update the license as specified on this JIRA: https://issues.jboss.org/browse/JDF-285

Thank you

**Comment by marcobattaglia** [comment]
Hi Rafael,
New version is here:
https://github.com/marcobattaglia/jboss-bom/tree/jboss-javaee-6.0-with-drools

I have  this doubt: 
- In my quickstart pom I have added under dependencyManagement this dependency:
  <dependencyManagement>......
                  <dependency>
              <groupId>org.jboss.bom</groupId>
              <artifactId>jboss-javaee-6.0-with-drools</artifactId>
              <version>${version.jboss.bom}</version>
              <type>pom</type>
              <scope>import</scope>
          </dependency>
  ....
  </dependencyManagement>
  But if I delete all dependencies related to drools under dependecies section it doesn't work. I can only delete version info of every drools dependencies, because this info is read from bom installed under my local .m2 repo. (I've typed mvn clean install and after I've used mvn -u ... starting my project)
  So the dependencies section of quickstart's pom now is:
  ......
  </dependencyManagement>
  <dependencies>
            <dependency>
          <groupId>org.drools</groupId>
          <artifactId>knowledge-api</artifactId>
             
      </dependency>
      <dependency>
          <groupId>org.drools</groupId>
          <artifactId>drools-core</artifactId>
      </dependency>
      <dependency>
          <groupId>org.drools</groupId>
          <artifactId>drools-compiler</artifactId>
      </dependency>
      <dependency>
          <groupId>org.drools.planner</groupId>
          <artifactId>drools-planner-core</artifactId>
      </dependency>
             <dependency>
          <groupId>org.apache.deltaspike.core</groupId>
          <artifactId>deltaspike-core-api</artifactId>
      </dependency>
  ......
  </dependencies>
  I don't know if it is right. I expected to delete the previous four dependencies as they are fully load by bom declared in dependencyManagement section, not only to delete version info.

Thank you

**Comment by marcobattaglia** [comment]
Sorry Rafael formatted text is been interpreted.
You cannot read actual pom of the quickstart project. 
Here it is:
https://github.com/marcobattaglia/jboss-as-quickstart/blob/master/cdi-portable-extension-drools/pom.xml

**Comment by rafabene** [comment]
Hi.

Sorry for the late answer but I was out for vacation. I look your PR and it's good to merge. The answer for you doubt is the following: It's right to work with the dependencies declared without the version. That's the purpose of the BOM. The dependenciManagement specifies the versions about each dependency, not the dependencies itself. :) Was that clear. 

Btw, Good job! Thank you.

**Comment by rafabene** [comment]
Merged

**Pull request closed without merging**